src/store/dashboard/ociDashboard/ociDashboardCommon.ts

```diff
@@ -30,7 +30,7 @@ export function getGroupByForTab(widget: OciDashboardWidget): OciQuery['group_by
       };
```

Comment:
I'm not sure we want to do this for OCI as this may have been an Azure specific issue.

Thinking we should just return this code below, unless we discover an issue with the API request for this overview tab?

```
case OciDashboardTab.product_services:
      return { product_service: '*', };
```